IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -949,6 +1090,17 @@ private void LoadImage(byte[] Bytes)
             {
                 throw new DllNotFoundException("Please install SixLabors.ImageSharp from NuGet.", e);
             }
+            catch (NotSupportedException e)
+            {
+                try
+                {
+                    OpenTiffToImageSharp(Bytes);
+                }
+                catch
+                {
+                    throw new NotSupportedException("Image cannot be loaded. File format doesn't supported.", e);
```

Comment:
Change:
`                    throw new NotSupportedException("Image cannot be loaded. File format doesn't supported.", e);`
To:
`                    throw new NotSupportedException("Image could not be loaded. File format is not supported.", e);`